cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -185,6 +187,13 @@ export default class Home extends Component {
           <Route path="/ns/:namespace/kitchen" component={ConfigurationGroupKitchenSync} />
           <Route path="/ns/:namespace/replication" component={Replicator} />
           <Route path="/ns/:namespace/lab" component={Lab} />
+          <Route
+            exact
+            path="/ns/:namespace/vs"
+            render={() => {
+              return <VirtualScrollDemo />;
```

Comment:
Why is it call demo? Is it really just for demo or is actual prod code?

Reply:
No this is an experimental page. The `VirtualScrollDemo` component is the page that renders the virtual scroll list. This is a temporary route where this experiment is parked.